### PR TITLE
Add non-hand-drawn figures to Chapters 1-10

### PR DESCRIPTION
**Pull request by pavpanchekha**
These are all the non-hand-drawn figures for Chapters 1-10. The hand-drawn ones, annoyingly, require me to upgrade to macOS 14 (what?!) but they are all drawn.

Also deleted one obsolete figure that we no longer use.

### CODE REVIEW

**Comment by pavpanchekha** [comment]
Let me also note that the way images look on the web version is currently not 100% great but I think that can be fixed in, like, a week. The images also don't look amazing in the LaTeX version but a lot of that will be up to the typesetter anyway.

File: book/html.md [hunk 1]
```diff
@@ -16,7 +16,18 @@ A tree of nodes
...
-and for each span of text.[^1] So for our browser to use a tree, tokens need to
+and for each span of text.[^1]
+
+[^1]: In reality there are other types of nodes too, like comments,
+    doctypes, and `CDATA` sections, and processing instructions. There
+    are even some deprecated types!
+
+::: {.print-only .center}
+![An annotated HTML document](im/html-syntax.png)
+^[An HTML document, showing tags, text, and the nesting structure.]
```
Comment (chrishtr) [inline book/html.md:27]: Why the footnote? Why not just rely on the alt text?

File: book/text.md [hunk 2]
```diff
@@ -39,6 +39,23 @@ typing. Variations—like bold or italic letters—were called that type's
 
 [^2]: The word is related to *foundry*, which would create the little
     metal shapes.
+    
+::: center
```
Comment (chrishtr) [inline book/text.md:43]:
```suggestion
::: {.center}
```

**Review by chrishtr — APPROVED** [review]

File: book/text.md (hunk 2 above)
Comment (pavpanchekha) [reply book/text.md:43]: Actually this works but you're right, we typically do this this way.

File: book/html.md (hunk 1 above)
Comment (pavpanchekha) [reply book/html.md:27]: Yeah, uhh, I don't know. This is how we do it in `history.md` and it looks terrible. I figured now wasn't the time to figure it out :(

**Pull request merged**